pandas-stubs/core/indexing.pyi

```diff
@@ -15,7 +15,7 @@ from pandas._typing import (
 )
 
 _IndexSliceTuple: TypeAlias = tuple[
-    Union[Index, MaskType, Scalar, list[ScalarT], slice], ...
+    Union[Index, MaskType, Scalar, list[ScalarT], slice | tuple[ScalarT, ...]], ...
```

Comment:
Same here (I think we need it for `list` but not sure about `tuple`).

Reply:
> Same here (I think we need it for `list` but not sure about `tuple`).

Yes, fixed in commit https://github.com/pandas-dev/pandas-stubs/pull/494/commits/3417725f9e174a36595017559c1f3e1c2419e807